setup.py

```diff
@@ -24,26 +25,35 @@
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
     "Topic :: Scientific/Engineering :: Mathematics"
 ]
 
-extensions = [Extension('quadprog', [
-    'quadprog/linear-algebra.c',
-    'quadprog/qr-update.c',
-    'quadprog/quadprog.c',
-    'quadprog/solve.QP.c',
-])]
+ext_modules=cythonize(
+    [
+        Extension(
+            "quadprog",
+            [
+                'quadprog/linear-algebra.c',
+                'quadprog/qr-update.c',
+                'quadprog/solve.QP.c',
+                'quadprog/quadprog.pyx',
+            ],
+        )
+    ],
+)
 
 setup(
-    name="quadprog",
+    name="quadprog-wheel",
     version="0.1.11",
-    description="Quadratic Programming Solver",
+    description="Quadratic Programming Solver with wheel packages",
     long_description=long_description,
     url="https://github.com/quadprog/quadprog",
     author="Robert T. McGibbon",
     author_email='rmcgibbo@gmail.com',
     license='GPLv2+',
     classifiers=classifiers,
-    ext_modules=extensions,
-    install_requires=["numpy"]
+    ext_modules=ext_modules,
+    python_requires=">=3.6, <3.11",
```

Comment:
Is the new ``python_requires`` required for the PR to work?

Although there is no classifier for it I believe ``quadprog`` can still be installed with Python 2.